### DISCUSSION TITLE
Para cuando para versión 1.19?

### DISCUSSION THREAD

**Discussion by wilebaldo** [JohanVonElectrum/Johan-Carpet#67] [Ideas]
Hola Johan, llegue a tu mod por recomendación de los videos de Rush, el comento que en tu mod tiene una función para reparar las herramientas con la barra de experiencia, me imagino que ya estas trabajando en ello para ponerlo en la versión 1.19 porque ya vi que va a ser independiente del carpet mod.

te tengo una pregunta, me gustaría saber cuales son las restricciones o permisos para hacer modificaciones del mod, no soy programador hace mucho tiempo que no lo hago, pero me gustaría nada mas por diversión entrar otra vez a hacer un par de cosas ahorita que estoy metido en el minecraft, y nada mas para aprender me gustaría hacer como el mod solo y simple nada mas con esa función que estoy buscando.

solo quiero saber que si puedo y que no puedo hacer con tu código.

**Comment by JohanVonElectrum** [comment]
**Hola, te contesto por partes.**


> me gustaría saber cuales son las restricciones o permisos para hacer modificaciones del mod

_Puedes leer el archivo LICENSE, pero como resumen, mientrás des crédito y referencies el original todo está bien._
> me imagino que ya estas trabajando en ello para ponerlo en la versión 1.19

_Aún tardaré, he estado ocupado, pero ahora que vuelvo me voy a centrar en 1.17.1._
> independiente del carpet mod

_Tengo ganas de empezar un proyecto así, pero tengo que sacar tiempo._


**Gracias por el interés.**
